examples/datepicker/datepicker-combobox.html

```diff
@@ -0,0 +1,945 @@
+    <div id="example">
+
+      <div id="myDatepicker"
+           class="datepicker"
+           role="combobox"
+           aria-expanded="false"
+           aria-haspopup="dialog"
+           aria-owns="myDatepickerDialog"
+           aria-labelledby="id-label-1">
+
+        <div class="date">
+          <label id="id-label-1"
+                 for="id-textbox-1">Date</label>
+
+          <input type="text"
+                placeholder="mm/dd/yyyy"
+                id="id-textbox-1"
+                aria-autocomplete="none"
+                aria-controls="id-datepicker-1">
+
+          <span class="arrow up">
+            <svg width="18" height="18">
+              <polygon points="4,16 14,16 9,9" style="fill:gray;stroke:gray;stroke-width:1" />
+            </svg>
+          </span>
+
+          <span class="arrow down">
+            <svg width="18" height="18">
+              <polygon points="4,9 14,9 9,16" style="fill:gray;stroke:gray;stroke-width:1" />
+            </svg>
+          </span>
+
+          <button class="icon"
+            tabIndex="-1"
+            aria-label="Change Date"
+            aria-haspopup="true"
+            aria-expanded="false"
+            title="Select to change date"
+            >
+            <span class="fa fa-calendar-alt fa-2x"></span>
+          </button>
+
+        </div>
+
+        <div class="message" role="status"></div>
+
+        <div id="myDatepickerDialog"
+             class="datepickerDialog"
+             role="dialog"
+             aria-labelledby="id-dialog-label">
+
+          <div class="header">
+
+            <button class="prevYear"
+              aria-label="previous year"
+              aria-controls="id-dialog-label">
+              <span class="fas fa-angle-double-left fa-lg"></span>
+            </button>
+
+            <button class="prevMonth"
+              aria-label="previous month"
+              aria-controls="id-dialog-label">
+              <span class="fas fa-angle-left fa-lg"></span>
+            </button>
+
+            <span id="id-dialog-label"
+                  class="monthYear"
+                  role="status">
+              December 2018
+            </span>
+
+            <button class="nextMonth"
+              aria-label="next month"
+              aria-controls="id-dialog-label">
+              <span class="fas fa-angle-right fa-lg"></span>
+            </button>
+
+            <button class="nextYear"
+              aria-label="next year"
+              aria-controls="id-dialog-label">
+              <span class="fas fa-angle-double-right fa-lg"></span>
+            </button>
+
+          </div>
+
+          <table
+            id="myDatepickerGrid"
+            class="dates"
+            role="grid"
+            aria-labelledby="id-dialog-label">
+            <thead>
+              <tr>
+                <th scope="col" aria-label="Sunday">Su</th>
```

Comment:
Is aria-label on a column header expected to affect the calculated column header name for individual grid cells? E.g. should navigating a table in a screen reader's scan mode read "Sunday" or "Su" when moving to a cell in this column? I'm getting pretty different results when testing.

Reply:
It should read "Sunday" according to the [HTML-AAM section on tr, td, th elements](https://w3c.github.io/html-aam/#tr-td-th-elements-accessible-name-computation).
Which AT/browser pairing(s) are reading "Su" for you? Might need to open some issues...

Reply:
HTML-AAM describes the accessible name for the header cell itself. I don't see anything in it which indicates what the header name is when reading an associated cell in the table. 
If the section in HTML-AAM is intended to cover that then it is missing the abbr attribute which should certainly be a part of that calculation.
Indeed it may be better to use abbr in this example so this would become
 ```
<th scope="col" abbr="Sunday">Su</th>
```

Reply:
> I don't see anything in it which indicates what the header name is when reading an associated cell in the table.

Good point.

> `<th scope="col" abbr="Sunday">Su</th>`

Interesting - I did not know about abbr _attribute_ on th. It's exactly what is needed here.
From the [HTML spec for th](http://w3c.github.io/html/tabular-data.html#the-th-element):
> The th element may have an abbr content attribute specified. Its value must be an alternative label for the header cell, to be used when referencing the cell in other contexts (e.g., when describing the header cells that apply to a data cell). It is typically an abbreviated form of the full header cell, but can also be an expansion, or merely a different phrasing.

I see it in the [HTML-AAM Attribute Table for th attr](https://w3c.github.io/html-aam/#att-abbr), but I don't really understand what it is saying.

Reply:
@scottaohara Any idea what  https://w3c.github.io/html-aam/#att-abbr means?

Reply:
changed `<th scope="col" aria-label="` to `<th scope="col" abbr"`